src/components/Navbar.vue

```diff
@@ -1,27 +1,32 @@
 <template>
-    <nav class="navbar">
+    <nav v-if="showNavbar" class="navbar">
```

Comment:
showNavbar, is that an example of a reactive variable? Is that why we use ref()?